### PR TITLE
add slides for 2017/10/17 meetup

### PR DESCRIPTION
**Pull request by patrick-blom**

### CODE REVIEW

File: phpugms_greeting_2017-10/index.html [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+                </section>
+
+
+                <section class="l-2up">
+                    <h2>November Meeting</h2>
+
+                    <strong>Tuesday, November 21, 2017</strong><br>
+
+                    <p>Holger Woltersdorf (@hollodotme )</p>
+                    <p>"Async requests & reactive responses with php-fpm"</p>
```
Comment (bcremer) [inline phpugms_greeting_2017-10/index.html]: `&` should be escaped.

File: phpugms_greeting_2017-10/index.html [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+                </section>
+
+                <section>
+                    <img src="img/cleancodelingen.png" height="650">
```
Comment (bcremer) [inline phpugms_greeting_2017-10/index.html]: Should be placed in the events section.

File: phpugms_greeting_2017-10/index.html [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+
+                <section class="l-2up">
+                    <h2>November Meeting</h2>
+
+                    <strong>Tuesday, November 21, 2017</strong><br>
+
+                    <p>Holger Woltersdorf (@hollodotme )</p>
+                    <p>"Async requests & reactive responses with php-fpm"</p>
+                </section>
```
Comment (bcremer) [inline phpugms_greeting_2017-10/index.html]: Holger will also have a second talk "IceHawk - a PHP 7 micro framework respecting CQRS"

File: README.md [hunk 4]
```diff
@@ -5,3 +5,7 @@
 ### September 20th 2017
 
 * [Introduction](http://phpugms.github.io/phpugms_greeting_2017-09/index.html)
+
+### October 17th 2017
```
Comment (bcremer) [inline README.md]: Can you please put that above the september meetup so we have the most recent meetup at the beginning.

File: phpugms_greeting_2017-10/index.html [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+                
+
+<section>
+    <h1>Upcoming Events</h1>
+</section>
+
+
+
+                <section>
+                    <img src="img/swhack.png">
```
Comment (bcremer) [inline phpugms_greeting_2017-10/index.html:95]: Nice. Are you attending btw?

**Comment by bcremer** [comment]
Thank you @patrick-blom.

**Comment by bcremer** [comment]
Looks good for me 👍 Feel free to merge.

File: phpugms_greeting_2017-10/index.html (hunk 5 above)
Comment (patrick-blom) [reply phpugms_greeting_2017-10/index.html:95]: Its marked in my calendar, but i still not have signed up ^^

File: phpugms_greeting_2017-10/index.html (hunk 1 above)
Comment (patrick-blom) [reply phpugms_greeting_2017-10/index.html]: fixed

File: phpugms_greeting_2017-10/index.html (hunk 2 above)
Comment (patrick-blom) [reply phpugms_greeting_2017-10/index.html]: fixed

File: phpugms_greeting_2017-10/index.html (hunk 3 above)
Comment (patrick-blom) [reply phpugms_greeting_2017-10/index.html]: also fixed ^^

File: README.md (hunk 4 above)
Comment (patrick-blom) [reply README.md]: fixed

**Pull request merged**